misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/entity/TesterinaFunction.java

```diff
@@ -117,19 +116,12 @@ private static Object runOnSchedule(Class<?> initClazz, String name, Scheduler s
                                                    null, null);
```

Comment:
Let's return the exception from https://github.com/ballerina-platform/ballerina-lang/pull/29883/files#diff-84a9a5920b825db97129fd62b5af503df886a08f4d41973222699c0a5db86570R112 as well